mockgcp/tools/gapic/pkg/sorted/map.go

```diff
@@ -51,16 +51,6 @@ func (m *Map[K, V]) Entries() []Entry[K, V] {
 	return m.entries
 }
 
-func (m *Map[K, V]) Get(k K) V {
```

Comment:
This function seems to be good, shall we keep it?

Reply:
It was duplicated from a few lines up.  I assume we fixed it in another PR.  I'll rebase!